fastly.toml

```diff
@@ -3,3 +3,11 @@ name = "Static content"
 description = "Apply performance, security and usability upgrades to static bucket services such as Google Cloud Storage or AWS S3."
 authors = ["<oss@fastly.com>"]
 language = "rust"
+
+[setup]
```

Comment:
@kailan is it possible to update the README to reflect that there is a `[setup]` configuration now.

Reply:
This makes sense. Will do!